web_digital_sign/static/src/css/jquery-ui.css

```diff
@@ -0,0 +1,1225 @@
+/*! jQuery UI - v1.11.2 - 2014-10-16
```

Comment:
Why do you need this? Isn't jQuery already bundled with web client?

Anyways, this is too heavy for only a couple of styles you surely need, don't you think?